target/linux/ramips/dts/OMEGA2.dtsi

```diff
@@ -0,0 +1,182 @@
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "onion,omega", "mediatek,mt7628an-soc";
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	bootstrap {
```

Comment:
this looks like the same copy/paste leftover I've already mentioned in https://github.com/lede-project/source/pull/645.